js/app/historyStore.js

```diff
@@ -262,7 +262,7 @@ const HistoryStore = new Lang.Class({
     // the same after a link click, factoring out this common function. When we
     // diverge in future interactions we should revisit this decomposition.
     show_ekn_id: function (ekn_id) {
-        Engine.get_default().get_object_by_id(ekn_id, null, (engine, task) => {
+        Engine.get_default().get_object_by_id(ekn_id, null, null, (engine, task) => {
```

Comment:
Not sure if you want this, as it seems pretty confusing to me, but spidermonkey 31 now allows parameters with default values to come before parameters without them: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters